Assets/RGLUnityPlugin/Scripts/LidarSnowManager.cs

```diff
@@ -54,6 +54,11 @@ public class LidarSnowManager : MonoBehaviour
         [field: Tooltip("If true, a more sophisticated method is used, which takes into account the energy loss of the lidar beam when hitting snowflakes")]
         public bool DoSimulateEnergyLoss { get; private set; } = true;
 
+        [field: SerializeField]
+        [field: Tooltip("Minimal snowflake occupancy (in percent) of the ray beam to be included in energy loss calculation")]
```

Comment:
"Minimal snowflake occupancy (in fraction of ray beam angle) included in energy loss calculation")]